index.html

```diff
@@ -7,15 +7,133 @@
+        <nav class="header__nav">
+          <a class="header__link header__link--active" href="/">Главная</a>
+          <a class="header__link" href="/">Информация</a>
+          <a class="header__link" href="/">Новости</a>
+          <a class="header__link" href="/">Контакты</a>
+        </nav>
+        <div class="header__profile">
+          <span class="header__profile-name">Ваше имя</span>
+          <img class="header__profile-img" src="/Avatar.png" alt="Изображение профиля" />
+        </div>
+      </div>
+    </header>
+
     <div class="container">
-      <h1 style="text-align: center">ПУСТОЙ ПРОЕКТ</h1>
+      <h1 class="main-title">Личная страница</h1>
+
+      <section class="personal-page">
+        <h2 class="personal-page__info">Полезная информация</h2>
+
+        <div class="personal-page__grid">
+          <div class="personal-page__block personal-page__block--yellow-background">
+            <h3 class="personal-page__list-title">Заголовок</h3>
+            <ul class="personal-page__list">
+              <li>Список 1</li>
+              <li>Список 2</li>
+              <li>Список 3</li>
+              <li>Список 4</li>
+            </ul>
+          </div>
+
+          <div class="personal-page__block">
+            <h3 class="personal-page__main-title">СФУ и Российский университет спорта стали партнёрами</h3>
+
+            <div>
+              <h4 class="personal-page__subtitle">Заголовок</h4>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+                deserunt mollit anim id est laborum.
+              </p>
+            </div>
+
+            <div>
+              <h4 class="personal-page__subtitle">Заголовок</h4>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+                ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+                deserunt mollit anim id est laborum.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="interesting-articles">
+        <h2 class="interesting-articles__title">Интересные статьи</h2>
+        <h3 class="interesting-articles__description">Описать, о чем эти статьи, кратко</h3>
```

Comment:
```suggestion
        <p class="interesting-articles__description">Описать, о чем эти статьи, кратко</p>
```
это не h3, а описание, поэтому в p